content/resources/data.toml

```
@@ -10,12 +10,30 @@ description = "A curated list of Rust code and resources, inspired by other awes
 categories = ["lists"]
 homepage_url = "//github.com/kud1ing/awesome-rust"
 
+[[items]]
+name = "Awesome wgpu"
+description = "A curated list of wgpu code and resources"
+categories = ["lists"]
+homepage_url = "github.com/rofrol/awesome-wgpu"
+
 [[items]]
 name = "Developing a game with Rust beginners"
 description = "Some thoughts on tools and libraries"
 categories = ["articles"]
 homepage_url = "//lukaskalbertodt.github.io/2016/09/01/developing-a-game-with-Rust-beginners-some-thoughts-on-tools-and-libraries.html"
 
+[[items]]
+name = "Game Server in 150 lines of Rust"
+description = "This tutorial will show you how to write asimple game server"
+categories = ["tutorials"]
+homepage_url = "//medium.com/@buterajay/game-server-in-150-lines-of-rust-ce1782199907"
+
+[[items]]
+name = "Graphics by Squares"
+description = "A gfx-rs tutorial"
+categories = ["tutorials"]
+homepage_url = "//https://suhr.github.io/gsgt"
```

This works, but it's a little confusing to have `//` *and* `https://`. You only need one :)

```suggestion
homepage_url = "//suhr.github.io/gsgt"
```